.github/workflows/tests.yml

```diff
@@ -0,0 +1,201 @@
+# Copyright (c) 2021 Concurrent Technologies Corporation.
+#                                                                                                               
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at                                                    
+#                                                                                                               
+#     http://www.apache.org/licenses/LICENSE-2.0                                                                
+#                                                                                                               
+# Unless required by applicable law or agreed to in writing, software is distributed under the License is       
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or              
+# implied.  See the License for the specific language governing permissions and limitations under the License.  
+
+---
+name: Unit Tests
+on:
+  push:
+    branches:
+      - '**'
+
+jobs:
+  build-lib-cpp-tests:
+    strategy:
+      matrix:
+        os: [ windows-latest, macos-latest, ubuntu-latest ]
+    name: Build ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Enable Developer Command Prompt 💻
+        if: runner.os == 'Windows'
+        uses: ilammy/msvc-dev-cmd@v1
+
+      - name: Checkout 🛎️
+        uses: actions/checkout@v3
+
+      - name: Setup Valgrind 🔬
+        if: runner.os == 'Linux'
+        run: sudo apt-get -y install valgrind
+
+      - name: Setup Java ☕
+        uses: actions/setup-java@v3.4.1
+        with:
+          distribution: temurin
+          java-version: 8
+          cache: sbt
+
+      - name: Setup Python 🐍
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+
+      - name: Install Conan 💪
+        id: conan
+        uses: turtlebrowser/get-conan@main
+
+      - name: Conan version 💪
+        run: echo "${{ steps.conan.outputs.version }}"
+
+      - name: Setup cmake 🔧
+        uses: lukka/get-cmake@latest
+
+      - name: Prepare, Build, and Install Ωedit 🔧
+        run: |
+          cmake -S . -B build -DCMAKE_BUILD_TYPE=Debug -DBUILD_SHARED_LIBS=YES --install-prefix "${PWD}/_install"
+          cmake --build build
+          cmake --install build/packaging --prefix "${PWD}/_install" --config Debug
+
+      - name: Copy library file mac/linux 🚚
+        if: runner.os != 'Windows'
+        run: cp -r _install/lib/* ./lib
```

Comment:
I don't think we need a recursive (-r) copy here, we just need the .dylib and the .so.